.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+.DS_Store
```

Comment:
I've missed that you've another .DS_Store file in `src/.DS_Store`. Please run `git rm --cached src/.DS_Store` to remove that.